app/experimenter/docs/adrs/0005-doc-hub.md

```diff
@@ -0,0 +1,232 @@
+- Some doc maintainers don't prefer checking into code and using GH¹
+- May pose a learning curve for those unfamiliar with Storybook, and out of all of our options, is arguably the most involved¹
+
+**Workflow for this option**
+
+Requires around `5` story points to set up, but more if other tools should be created for it. Doc maintainers could directly edit MD files in GitHub, but if a new page is needed, a corresponding JS/JSX file may be needed to display it in unless effort was put into a tool. Storybook more than any other option would ask maintainers to check into the code. Whether directly in GitHub or on the doc maintainer's machine, changes would be placed a pull request into Experimenter with a label for issue-label-notifier to pick up.
+
+A demo should likely be made to help ease the process for maintainers.
+
+#### mdBook
+
+[`mdBook`](https://github.com/rust-lang/mdBook) is a tool that generates book-like documentation from Markdown files. For this option, we would likely set up a `docs` or similar branch specifically for documentation in the Experimenter repo and we would point GH to push this branch to [GitHub pages](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).
+
+Pros:
+
+- Has a search bar that appears to be robust
+- Is used extensively by [Glean](https://docs.telemetry.mozilla.org/concepts/glean/glean.html) and is written in Rust
+- Deployment would be handled by GitHub Pages and the URL would be easily accessible
+
+Neutral:
+
+- Has a flexible site navigation in title links and sublinks, but does not support external links in the sidebar, though there is [an open issue](https://github.com/rust-lang/mdBook/issues/919) for the feature
+- Documentation edits would require a PR and approval in the Experimenter repo, and notifications for approval (and for awareness) could be subscribed to with a tool like [`issue-label-notifier`](https://github.com/marketplace/actions/issue-label-notifier)
+- While Docusaurus pulls in external MD files and builds the sidebar automatically (optionally customizable), `mdBook` requires an edit in the [`SUMMARY.MD`](https://rust-lang.github.io/mdBook/format/summary.html) file as well to reflect sidebar updates
+- This package was made for open source documentation, but does not appear to have plugins or add-ons
+- [opinion] Appears to be similar to Docusaurus without the option to write custom docs with JS when preferred
+
+Cons:
+
+- Requires some, albeit not a lot, of setup
+- Does not support external links in the sidebar, though there is [an open issue](https://github.com/rust-lang/mdBook/issues/919) for the feature
+- Is not as flexible as Docusaurus or Storybook and only allows MD files
+  - Multi-language code examples would be flat, take up more screen space, and visually not look as ideal as they could with a more flexible option like Docusaurus or Storybook
```

Comment:
Eh, multi-language code examples don't have to be "flat" and take up more screen space, we found a workaround for this in mdbook in Glean: see the code examples [here](https://mozilla.github.io/glean/book/user/metrics/boolean.html), for instance.

Reply:
Ohh nice, that's super helpful! I actually clicked around the Glean docs seeing if there was something similar and missed it. This might tip the scales away from Docusaurus.